common/config/rush/pnpm-lock.yaml

```diff
@@ -18940,15 +18946,16 @@ packages:
       rollup: 2.42.4
       ts-jest: 26.5.6_jest@26.6.0+typescript@4.3.5
       typescript: 4.3.5
+      underscore: 1.12.1
     dev: false
     name: '@rush-temp/calling-component-bindings'
     resolution:
-      integrity: sha512-yH3HaUlLpmJpBR9rV5P2I0ECibbmgkQrGYiKbrVGG2+UGAKmfJW3894+/w6PL4sj9bxi83uxtgWz6XDPrzHTfw==
+      integrity: sha512-x19zF8P7mTYvIADFg1brpB9ceIMcfaa6a+PSORQrV9URmhJMpVSDB0RddMtjS7c/TgKm0/z5lJ2rwGKgsn8C3g==
       tarball: file:projects/calling-component-bindings.tgz
     version: 0.0.0
   file:projects/calling-stateful-client.tgz:
     dependencies:
-      '@azure/communication-calling': 1.4.2-beta.1
+      '@azure/communication-calling': 1.3.2
```

Comment:
This looks incorrect, but not sure why the non-stable is saying it is using the stable dependency,
cc @PorterNan